docs/user/exams/instructors_guide.rst

```diff
@@ -493,11 +493,15 @@ You can specify the moment when Artemis publishes the results of the exam, see `
     Unsubmitted exams are not eligible for the assessment process.
 
 - Review student performance using various metrics such as average, median and standard deviation.
-- Unsubmitted exams are not eligible for assessment and thereby appear as having no score. It can happen that an exercise is not part of any :ref:`student exam <student_exams>`. This is the case when Artemis selects a different exercise of the same exercise group for every :ref:`student exam <student_exams>`. Similarly to the unsubmitted exams, they can warp the results and statistics of the exam. By eliminating unsubmitted exams and exercises which were not part of the :ref:`exam conduction <exam_conduction>`, you can gain a more realistic overview of the performance of the students.
+- Unsubmitted exams are not eligible for assessment and thereby appear as having no score and the corresponding students are assigned with :ref:`no-participation special grade <grading>` if a grading key exists. It can happen that an exercise is not part of any :ref:`student exam <student_exams>`. This is the case when Artemis selects a different exercise of the same exercise group for every :ref:`student exam <student_exams>`. Similarly to the unsubmitted exams, they can warp the results and statistics of the exam. By eliminating unsubmitted exams and exercises which were not part of the :ref:`exam conduction <exam_conduction>`, you can gain a more realistic overview of the performance of the students.
```

Comment:
```suggestion
- Unsubmitted exams are not eligible for assessment and thereby appear as having no score. The corresponding students are assigned with a :ref:`no-participation special grade <grading>` if a grading key exists. It can happen that an exercise is not part of any :ref:`student exam <student_exams>`. This is the case when Artemis selects a different exercise of the same exercise group for every :ref:`student exam <student_exams>`. Similarly to the unsubmitted exams, they can warp the results and statistics of the exam. By eliminating unsubmitted exams and exercises which were not part of the :ref:`exam conduction <exam_conduction>`, you can gain a more realistic overview of the performance of the students.
```